packages/react-dom/src/__tests__/ReactDOMTextarea-test.js

```diff
@@ -423,4 +425,126 @@ describe('ReactDOMTextarea', () => {
     ReactDOM.unmountComponentAtNode(container);
     ReactDOM.render(<textarea value={undefined} />, container);
   });
+
+  describe('When given a Symbol value', () => {
+    it('treats initial Symbol value as an empty string', () => {
+      const container = document.createElement('div');
+      expect(() =>
+        ReactDOM.render(
+          <textarea value={Symbol('foobar')} onChange={() => {}} />,
+          container,
+        ),
+      ).toWarnDev('Invalid value for prop `value`');
+      const node = container.firstChild;
+
+      expect(node.value).toBe('');
+    });
+
+    it('treats initial Symbol children as an empty string', () => {
+      const container = document.createElement('div');
+      expect(() =>
+        ReactDOM.render(
+          <textarea onChange={() => {}}>{Symbol('foo')}</textarea>,
+          container,
+        ),
+      ).toWarnDev('Use the `defaultValue` or `value` props');
+      const node = container.firstChild;
+
+      expect(node.value).toBe('');
+    });
+
+    it('treats updated Symbol value as an empty string', () => {
+      const container = document.createElement('div');
+      ReactDOM.render(<textarea value="foo" onChange={() => {}} />, container);
+      expect(() =>
+        ReactDOM.render(
+          <textarea value={Symbol('foo')} onChange={() => {}} />,
+          container,
+        ),
+      ).toWarnDev('Invalid value for prop `value`');
+      const node = container.firstChild;
+
+      expect(node.value).toBe('');
+    });
+
+    it('treats initial Symbol defaultValue as an empty string', () => {
+      const container = document.createElement('div');
+      ReactDOM.render(<textarea defaultValue={Symbol('foobar')} />, container);
+      const node = container.firstChild;
+
+      // TODO: add warnings once defaultValue is compatible for them
```

Comment:
I don't quite understand this comment. @nhunzaker I think this is copied over from #11741 - Can you point me to something so I can read up on that?

Reply:
Sorry, should've been more thorough. I've added this comment as a reminder for future references once this [PR](https://github.com/facebook/react/pull/13360) is good to go. Currently there are no warnings for [reserved props](https://github.com/facebook/react/blob/3cfab14b96a55d53ba5784a9915b8c1660da08be/packages/react-dom/src/shared/DOMProperty.js#L206) in React and hence the TODOs in @nhunzaker's PR and this one :-)

Reply:
I know I'm the first offender here (what is ` // TODO: we should warn here.` even? 😿 ), but could you change this comment to something like:

```
// TODO: defaultValue is a reserved prop and is not validated. Check warnings when they are.
```